docs/tutorials/mastering.md

```diff
@@ -76,9 +695,68 @@ sudo microk8s.config > control-plane-cluster.yaml
 scp control-plane-cluster.yaml juju-controller.mgmt:
 ```
 
-### SD-Core User Plane
+Set alias for kubectl:
+```console
+cat << EOF | sudo tee -a ~/.bashrc
+alias kubectl="microk8s.kubectl"
+EOF
+source ~/.bashrc
+```
+
+Change coredns configmap by adding local dns server which is `10.201.0.100`.
```

Comment:
What about `sudo microk8s enable dns:10.201.0.100`?

Reply:
Configuring 10.201.0.100 as primary dns does not work for us. Our local dns server should be configured as domain based, otherwise pods can not solve external domain names.

Reply:
dnsmasq is a caching DNS server, so it will resolve based on what upstream DNS server you configure.  Or perhaps I don't understand what you mean by it can not solve external domain names?

Reply:
If we enable microk8s dns, 8.8.8.8 8.8.4.4 are default nameservers. However, if we run `sudo microk8s enable dns:10.201.0.100`, 10.201.0.100 will be the default nameserver. In that case, pods can not pull the images as they are not resolved properly.
```
data:
  Corefile: |
    .:53 {
        errors
        health {
          lameduck 5s
        }
        ready
        log . {
          class error
        }
        kubernetes cluster.local in-addr.arpa ip6.arpa {
          pods insecure
          fallthrough in-addr.arpa ip6.arpa
        }
        prometheus :9153
        forward . 8.8.8.8 8.8.4.4
        cache 30
        loop
        reload
        loadbalance
    }
    ```

Reply:
I still fail to understand why we need to specify a different DNS server as I've never had to do this in any of my testing.

Reply:
Could you please send me the content of the /etc/dnsmasq.conf file in your dns server ?

Reply:
no-resolv
server=8.8.8.8
server=8.8.4.4
domain=mgmt

Reply:
Although I set Google DNS servers in the config file, if our local dns is the default one, it fails to resolve external domain names such as (charmhub, dockerhub, etc.)

Reply:
That is the reason I do keep the default nameservers as Google DNS servers and set the local dns server just for mgmt domain.